src/transformers/models/layoutlm/modeling_layoutlm.py

```diff
@@ -774,8 +775,6 @@ def forward(
         >>> outputs = model(
         ...     input_ids=input_ids, bbox=bbox, attention_mask=attention_mask, token_type_ids=token_type_ids
         ... )
-
-        >>> last_hidden_states = outputs.last_hidden_state
```

Comment:
Why is this removed?

Reply:
Thanks for catching. It's an unfortunate side-effect of me running the docstring tests (and then "undoing" the changes the test makes). I'll revert these changes.